website/docs/result_to_dict.md

```diff
@@ -0,0 +1,72 @@
+---
+id: result_to_dict
+title: Result to Dictionary
+sidebar_label: Result to Dictionary
+---
+## ResultToDict Decorator
+
+The `ResultToDict` decorator will format any value a function will return.
+For eg. will return tuple as a tuple, dict as a dict.
+
+```python
+class ResultToDict(object):
+
+    def __init__(self, callback: Callable[[dict], Awaitable[dict]] = None):
+        self.callback = callback
+
+    def __call__(self, func, *args, **kwargs):
+
+        @wraps(func)
+        def __wrapper(*args, **kwargs):
+            return_val = func(*args, **kwargs)
+            return result_to_dict(return_val, properties_as_dict=True, callback=self.callback)
+        return __wrapper
+
+```
+

```

Comment:
as i made a example before 
extract each parameter and explain what it is doing